src/api/config/options.yml.example

```diff
@@ -26,11 +26,6 @@ default: &default
   frontend_host: localhost
   frontend_port: 443
   frontend_protocol: https
-  # if your users access the hosts through a proxy (or just a different name, use this to
-  # overwrite the settings for users)
-  #external_frontend_host: api.opensuse.org
-  #external_frontend_port: 443
-  #external_frontend_protocol: https
```

Comment:
There are some other lines that could be removed from `dist/functions.setup-appliance.sh`:
```bash
# grep -r external_frontend_ dist/
dist/functions.setup-appliance.sh:      sed -i 's,^external_frontend_host: .*,frontend_host: "'"$FQHOSTNAME"'",' $api_options_yml
dist/functions.setup-appliance.sh:      sed -i 's,^external_frontend_port: .*,frontend_port: 443,' $api_options_yml
dist/functions.setup-appliance.sh:      sed -i 's,^external_frontend_protocol: .*,frontend_protocol: "'"https"'",' $api_options_yml
```

Reply:
:+1: done